### ISSUE TITLE
Are you is working?

### ISSUE THREAD

**Issue by M-Mortaz** [maria-j70/wallet#3]
https://github.com/maria-j70/wallet/blob/1e39e0a802f6f3c149965809294beb46823b821d/transaction/models.py#L57

This is new to me , I means database error. Are you sure this exception will be raise if tracker ID be duplicated? Have you any smoke test for it?

**Comment by maria-j70** [comment]
yes, i tested it because transaction model just have created_at and tracker_id and it will raise an error when tracker_id is dublicated
